### ISSUE TITLE
[feat] : Filtering Sidebar

### ISSUE THREAD

**Issue by bj-jiwrajka** [Open-Source-Chandigarh/Brewtopia#68]
It would be great if we can filter out products on the basis of price

**Comment by Varinder-Dhillon0** [comment]
hey @bj-jiwrajka i am thinking that we make a bar like this one on left
![image](https://github.com/Open-Source-Chandigarh/Brewtopia/assets/98728916/cc6997a5-1095-4083-b05c-abe85ca245f9)

and shift search bar in it after that we can add price bar to sort

**Comment by bj-jiwrajka** [comment]
> hey @bj-jiwrajka i am thinking that we make a bar like this one on left ![image](https://private-user-images.githubusercontent.com/98728916/294653417-cc6997a5-1095-4083-b05c-abe85ca245f9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ1MjA5ODcsIm5iZiI6MTcwNDUyMDY4NywicGF0aCI6Ii85ODcyODkxNi8yOTQ2NTM0MTctY2M2OTk3YTUtMTA5NS00MDgzLWIwNWMtYWJlODVjYTI0NWY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA2VDA1NTgwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlOGNiMDMzMGM5MDFmYzFjNzIzNzdmOTNiM2MzM2ZjODZkMmUyMDU4NTc4MDBjZTlhNjk5MTM3NzU3YjU1ZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M_uALA_84wruke9LYj7KzVKyTaB5l31IhgJz_J6RNgI)
> 
> and shift search bar in it after that we can add price bar to sort

okay, I'll do that

**Comment by Varinder-Dhillon0** [comment]
Are you still working on this @bj-jiwrajka

**Comment by hanzalahwaheed** [comment]
I'd like to work on it if available. Please assign the issue to me. Thanks!

**Comment by Varinder-Dhillon0** [comment]
ok @hanzalahwaheed

**Comment by hanzalahwaheed** [comment]
@Varinder-Dhillon0 hey im having some trouble in setting up the project. I didn't know elsewhere to seek help.
`const apiUrl = process.env.REACT_APP_API_URL;` 
i dont know if this has got to do with hosting the project. but im not able to send api requests without this value. what do I add in the .env file since it wasnt given in the README.md. waiting for your reply.

**Comment by Varinder-Dhillon0** [comment]
@hanzalahwaheed 

In client folder change .env.example name to .env and save file with this :-
REACT_APP_API_URL=https://brewtopia.up.railway.app/

**Comment by hanzalahwaheed** [comment]
@Varinder-Dhillon0 
finally got the project set up locally. you might wanna add the information in the README.md that you need to create a .env in the client with value `REACT_APP_API_URL=http://localhost:5000` and the .env in the server needs to have a value `ACCESS_URL=http://localhost:3000` 

only after this I was able to send api requests to access the home screen.
I could create a separate issue and a PR addressing this as well.
hope this helps :)

**Comment by Varinder-Dhillon0** [comment]
ok @hanzalahwaheed

**Comment by IrfanshaikH-7** [comment]
@hanzalahwaheed  are you working on this

**Comment by hanzalahwaheed** [comment]
> @hanzalahwaheed are you working on this

u can work on the sidebar feature if you want to

**Comment by IrfanshaikH-7** [comment]
Like !? Just the sidebar .or altogether with filtering ..

**Comment by hanzalahwaheed** [comment]
> Like !? Just the sidebar .or altogether with filtering ..

I think you will have to address the whole issue

**Comment by IrfanshaikH-7** [comment]
Sure ! I'll take over the issue - with your permission then..

**Comment by IrfanshaikH-7** [comment]
@Varinder-Dhillon0  could you assign me this so I could start working on it ..

**Comment by Varinder-Dhillon0** [comment]
yes @IrfanshaikH-7

**Comment by IrfanshaikH-7** [comment]
Allright - I'll start working on it -! It may take a little long please hold on till then - I'll make it ready in maximum of 2 days.

**Comment by Varinder-Dhillon0** [comment]
sure @IrfanshaikH-7
